armory/scenarios/poisoning_obj_det.py

```diff
@@ -0,0 +1,529 @@
+import copy
+import imgaug.augmenters as iaa
+from imgaug.augmentables.bbs import BoundingBox, BoundingBoxesOnImage
```

Comment:
In lieu of https://twosixtechnologies.slack.com/archives/G021C1YDUN5/p1682968354706919 it may be worth having a try catch block here since yoloattacks is not in the default dependencies list.

```suggestion
try:
    import imgaug.augmenters as iaa
    from imgaug.augmentables.bbs import BoundingBox, BoundingBoxesOnImage
except ImportError:
    raise ImportError(
        "imgaug is not installed. Please install via pip install armory-testbed[yolo]."
    )
```

Reply:
This won't be the only thing someone's missing if they didn't install the [yolo] version.  but it can't hurt